meson.build

```diff
@@ -0,0 +1,31 @@
+project('list-sensors', 'cpp',
+    default_options: [
+        'warning_level=3',
+        'werror=true',
+        'cpp_std=c++17',
+    ],
+    license: 'Apache-2.0',
+    version: '2.0',
+)
+
+conf = configuration_data()
+
+conf.set_quoted('MAPPER_BUS', get_option('mapper-bus'))
+conf.set_quoted('MAPPER_PATH', get_option('mapper-path'))
+conf.set_quoted('MAPPER_IFACE', get_option('mapper-iface'))
+conf.set_quoted('SYSTEMD_PROPERTIES', get_option('systemd-properties'))
+conf.set_quoted('SENSORS_PATH', get_option('sensors-path'))
+conf.set_quoted('SENSOR_VALUE_IFACE', get_option('sensor-value-iface'))
+
+conf.set('WITH_REMOTE_HOST', get_option('remote-host-support'))
+
+configure_file(output: 'config.h', configuration: conf)
+
+executable('lssensors',
+    'list-sensors.cpp',
+    dependencies: [
+        dependency('sdbusplus'),
+    ],
+    install: true,
+    install_dir: get_option('sbindir'),
```

Comment:
As I remember, OpenBMC community prefers bin, not sbin .

Reply:
exactly, but in reverse. ;)

Besides, as this tool requires root privileges to query D-Bus services, the `sbin` is the right place for it.

Reply:
All OpenBMC projects work with D-Bus, so we need to put them all in sbin ;)
Not so long ago, there are a lot of commits in OpenBMC, that fix the installation path and set it as /usr/bin.

Reply:
You are right.
https://github.com/openbmc/docs/blob/master/anti-patterns.md#placement-of-applications-in-sbin-or-usrsbin

I was sure that it is in reverse. ;) I will fix it.

Reply:
done